docs/modules/ROOT/partials/supported-versions.adoc

```diff
@@ -2,4 +2,4 @@
 // This is a separate file, since it is used by both the direct Druid documentation, and the overarching
 // Stackable Platform documentation.
 
-- 0.22.0
+- 0.22.1
```

Comment:
https://github.com/apache/druid/releases/tag/druid-0.22.1
I would say so, it has several security issues fixed.

Reply:
The version is backwards compatible to 0.22.0 and basically only includes updated dependencies. I think it might be worth thinking about whether we support older patch versions or not and come up with a general rule maybe? I added it to the architecture discussion agenda maybe we can discuss then

Reply:
Thank you. At the moment this is not important but in the future we can't just drop old versions because customers might still have them running. 
We need a policy on how long we support versions of our products.

Reply:
Yes I agree as well